packages/angular_devkit/build_ng_packagr/src/build/index.ts

```diff
@@ -5,80 +5,41 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import { BuilderContext, BuilderOutput, createBuilder } from '@angular-devkit/architect/src/index2';
+import { resolve } from 'path';
+import { Observable, from } from 'rxjs';
+import { catchError, mapTo, switchMap } from 'rxjs/operators';
+import { Schema as NgPackagrBuilderOptions } from './schema';
+
+async function initialize(
+  options: NgPackagrBuilderOptions,
+  root: string,
+): Promise<import ('ng-packagr').NgPackagr> {
```

Comment:
Since `ng-packagr` is a peerDep why not import it directly?

Reply:
Limits the volume of modules/packages that need to be loaded until the builder is actually executed.